Tensile/Source/lib/source/ContractionSolution.cpp

```diff
@@ -392,6 +392,11 @@ namespace Tensile
             rv.args.append<typename TypedInputs::BType const* const*>("batchB", inputs.batchB);
         }
 
+        rv.args.append<uint64_t>("offsetD", d.offset());
+        rv.args.append<uint64_t>("offsetC", c.offset());
+        rv.args.append<uint64_t>("offsetA", a.offset());
+        rv.args.append<uint64_t>("offsetB", b.offset());
```

Comment:
I can't say I've seen the negative offset or stride use case yet, but on our side these are all signed 64bit.  Maybe you just want to use int64_t and do signed arithmetic in the assembly?

Reply:
Thanks for checking that. I updated the arguments to int64_t, but since it is not a current use case, I think I'll defer updating the assembly math to a separate ticket.

Reply:
Yes that is fine we can defer as I pointed out we need to update rocblas side for this to match as we also stored in unsigned in the RocblasContractionProblem struct.   I commented on the commit below which is maybe invisible.  But if we defer this work you may want to keep the size_t for now?

Reply:
Changed it back to uint64_t for now and created a ticket to tackle signed offsets after these bug fixes are done